python/lsst/meas/algorithms/__init__.py

```diff
@@ -23,7 +23,26 @@
 """lsst.meas.algorithms
 """
 from __future__ import absolute_import
-from .algorithmsLib import *
+
+import lsst.afw.image
+import lsst.afw.math
+from lsst.afw.image import DefectBase
```

Comment:
Or should we use `py::module::import("lsst.afw.image._defect");`? Do we have policy for imports to use lowest level here? Of course this will have to be changed to `defect` once the `afw` cleanup is merged.